### DISCUSSION TITLE
port 10333

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#2271] [General]
hi hello 
some of my friends  they can create multiple team talk on port 10333
How do they do this?
thanks

**Comment by bear101** [comment]
You can do this if you have multiple IP-addresses. Then bind to the specific IP-address using the server's setup wizard.

**Reply by farshad-fallah** [reply]
thanks alot ok for example my second ip is 185.79.157.22
how I can run the wizard
with which command
